x-pack/test/security_api_integration/tests/session_concurrent_limit/global_limit.ts

```diff
@@ -212,6 +218,9 @@ export default function ({ getService }: FtrProviderContext) {
       // hence which specific sessions will be outside the limit.
       const statusCodes = [];
       for (const basicSessionCookie of basicSessionCookies) {
+        // This index refresh is only in the loop because get actually calls invalidate,
```

Comment:
__nit:__ `get` part sounds a bit confusing, maybe something a bit more wordy?
```suggestion
        // This index refresh is only in the loop because Kibana might internally call invalidate after it retrieved the session that's outside the limit
```